app/controllers/api_keys_controller.rb

```diff
@@ -0,0 +1,54 @@
+class ApiKeysController < ApplicationController
+  include ApiKeyable
+  before_action :redirect_to_signin, unless: :signed_in?
+
+  def index
+    @api_keys = current_user.api_keys
+    redirect_to new_profile_api_key_path if @api_keys.empty?
+  end
+
+  def new
+    @api_key = current_user.api_keys.build
+  end
+
+  def create
+    key = generate_unique_rubygems_key
+    @api_key = current_user.api_keys.build(api_key_params.merge(hashed_key: hashed_key(key)))
+
+    if @api_key.save
+      Mailer.delay.api_key_created(@api_key.id)
+
+      flash[:notice] = t(".save_key", key: key)
```

Comment:
I tested this PR on my dev machine and didn't notice that the API key was in the notification bar (which IMO is not great already) until i clicked to go to another page.

Can we instead make a panel above the API keys that shows this message instead, and with the API key specifically highlighted/bold

Reply:
![Screenshot from 2020-07-25 01-29-06](https://user-images.githubusercontent.com/7680662/88433424-a7084f80-ce1b-11ea-88e9-bddb7855d201.png)
I hope this works.